test/package/templates/fips/node.options

```diff
@@ -0,0 +1,4 @@
+--max-old-space-size=812
```

Comment:
was this intended to be `812` or `8192`?

Reply:
afaik we want a small heap allocation on these artifact pipelines to avoid OOM issues.

Reply:
understood, just 812 looked a bit too suspicious